contracts/core/Psm.sol

```diff
@@ -34,6 +34,7 @@ abstract contract PsmCore is IPSMcore, ModuleState, Context {
     function repurchase(Id id, uint256 amount)
         external
         override
+        PSMWithdrawalNotPaused(id)
```

Comment:
i think we better merge #100 into this branch for the correct modifier

Reply:
yeah